okhttp/src/jvmMain/kotlin/okhttp3/internal/cache/CacheInterceptor.kt

```diff
@@ -68,7 +68,6 @@ class CacheInterceptor(internal val cache: Cache?) : Interceptor {
           .protocol(Protocol.HTTP_1_1)
           .code(HTTP_GATEWAY_TIMEOUT)
           .message("Unsatisfiable Request (only-if-cached)")
-          .body(EMPTY_RESPONSE)
```

Comment:
This doesn't appear like a straight conversion, if the default is `EMPTY_BYTE_ARRAY.toResponseBody()`, doesn't this give the Response a body effectively?

Reply:
@swankjesse want to flag this question.

Reply:
Yeah the default is empty, not unreadable.